core/src/test/java/com/exadel/etoolbox/rolloutmanager/core/servlets/CollectLiveCopiesServletTest.java

```diff
@@ -181,7 +181,7 @@ void doPost_CollectItems_NonEmptyArrayResponse() throws WCMException, IOExceptio
         fixture.doPost(request, response);
 
         String expected = new String(Files.readAllBytes(Paths.get(EXPECTED_RESPONSE_JSON)))
-                .replaceAll("(\\r|\\n|\\t|\\s)", StringUtils.EMPTY);
+                .replaceAll("(\\r|\\n|\\t|\\s(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$))", StringUtils.EMPTY);
```

Comment:
This one looks a bit hard to read) Did you intend to remove spaces around the `:` symbol in `"foo": "bar"`? Why not do it manually in the resource file?)